release-please-config.json

```diff
@@ -0,0 +1,76 @@
+{
+  "separate-pull-requests": true,
+  "packages": {
+    "providers/openfeature-flagd-provider": {
+      "package-name": "openfeature-flagd-provider",
+      "version-file": "lib/openfeature/flagd/provider/version.rb",
+      "bump-minor-pre-major": true,
+      "bump-patch-for-minor-pre-major": true,
+      "versioning": "default",
+      "extra-files": ["README.md"]
+    },
+    "providers/openfeature-sdk-meta_provider": {
+      "package-name": "openfeature-flagd-provider",
```

Comment:
```suggestion
      "package-name": "openfeature-sdk-meta_provider",
```

Reply:
@maxveldink how would you feel about `openfeature-meta-provider` or something similar? I think we should drop the `sdk` portion.